docs/conf.py

```diff
@@ -28,14 +28,14 @@
 # -- Project information ------------------------------------------------------
 project = configuration["name"]
 author = f"{configuration['authors'][0]['name']} <{configuration['authors'][0]['email']}>"
-copyright = f"{datetime.datetime.now().year}, {configuration['authors'][0]['name']}"
+copyright = f"{datetime.datetime.now().year}, {author}"
 
 release = distribution(configuration["name"]).version
 # for example take major/minor
-version = ".".join(release.split(".")[:2])
+version = ".".join(release.split(".")[:3])
```

Comment:
```suggestion
version = ".".join(release.split(".")[:2])
```
Based on the sphinx docs I think this should stay as `:2` to yield `4.0`:
https://www.sphinx-doc.org/en/master/usage/configuration.html#confval-version